File: trunk/maven-support/maven-archetype-scalawebapp/src/main/resources/archetype-resources/src/main/scala/WelcomeServlet.scala
package ${groupId}

import java.io.IOException
import java.util.Date

import javax.servlet.ServletException
import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
import org.slf4j.{Logger, LoggerFactory}
	
class WelcomeServlet extends HttpServlet {
	val logger = LoggerFactory.getLogger(this.getClass)
	
	override def doGet(req:HttpServletRequest, resp:HttpServletResponse):Unit = {
		logger.debug("Received request for message")
		val serverTime = new Date()
		req.setAttribute("message", "Hello guess, the current server time is: " + serverTime)
		req.getRequestDispatcher("welcome.jsp").forward(req, resp)
	}	
}

File: tags/javasupport-0.0.3/maven-support/maven-archetype-webapp/src/main/resources/archetype-resources/src/main/java/WelcomeServlet.java
package ${groupId};

import java.io.IOException;
import java.util.Date;

import javax.servlet.ServletException;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
	
public class WelcomeServlet extends HttpServlet {
	protected Logger logger = LoggerFactory.getLogger(this.getClass());
	
	@Override
	protected void doGet(HttpServletRequest req, HttpServletResponse resp)
			throws ServletException, IOException {
		logger.debug("Received request for message");
		Date serverTime = new Date();
		req.setAttribute("message", "Hello guess, the current server time is: " + serverTime);
		req.getRequestDispatcher("welcome.jsp").forward(req, resp);
	}	
}
